server-api/src/main/java/org/sonarsource/sonarlint/core/serverapi/organization/OrganizationApi.java

```diff
@@ -36,25 +35,11 @@ public OrganizationApi(ServerApiHelper helper) {
   }
 
   public List<ServerOrganization> listUserOrganizations(ProgressMonitor progress) {
-    var serverChecker = new ServerVersionAndStatusChecker(helper);
-    return listUserOrganizations(serverChecker, progress);
+    return fetchUserOrganizations(progress);
   }
 
   public Optional<ServerOrganization> getOrganization(String organizationKey, ProgressMonitor progress) {
-    var serverChecker = new ServerVersionAndStatusChecker(helper);
-    checkServer(serverChecker, progress);
-    return fetchOrganization(organizationKey, progress.subProgress(0.2f, 1.0f, "Fetch organization"));
-  }
-
-  private List<ServerOrganization> listUserOrganizations(ServerVersionAndStatusChecker serverChecker, ProgressMonitor progress) {
-    checkServer(serverChecker, progress);
-    return fetchUserOrganizations(progress.subProgress(0.2f, 1.0f, "Fetch organizations"));
-  }
-
-  private static void checkServer(ServerVersionAndStatusChecker serverChecker, ProgressMonitor progress) {
-    progress.setProgressAndCheckCancel("Check server version", 0.1f);
-    serverChecker.checkVersionAndStatus();
-    progress.setProgressAndCheckCancel("Fetch organizations", 0.2f);
+    return fetchOrganization(organizationKey, progress);
```

Comment:
Nitpick: we can probably get rid of those 2 delegate methods (list and get)

Reply:
:+1: